src/maidsafe/vault/maid_manager/action_create_remove_account.h

```diff
@@ -96,9 +96,15 @@ std::string ActionCreateRemoveAccount<Remove>::Serialise() const {
   return action_proto.SerializeAsString();
 }
 
+#ifndef WIN32
```

Comment:
Yeah, that macro evaluates to the current compiler's major-minor numbers, which I suppose we'll always get away with, since 1700, as of now, is not likely to be defined elsewhere. I was going to use _WIN32 which is defined by the compiler but since WIN32 is defined by our project I used it instead. I meant to get back to you on this to ask since we're using compiler definitions what's the consensus on which to use, although by your comment I now know it's _MSC_VER. Thankss for merging and I'll change it at first opportunity.